front/src/common/AddNewCard.tsx

```diff
@@ -0,0 +1,49 @@
+import type { ReactNode } from 'react';
+
+import { useTranslation } from 'react-i18next';
+import { FaPlus } from 'react-icons/fa';
+
+import useUserRoleCheck from './authorization/hooks/useUserRoleCheck';
+import { REQUIRED_USER_ROLES_FOR } from './authorization/roleBaseAccessControl';
+import { useModal } from './BootstrapSNCF/ModalSNCF';
+
+type AddNewCardProps = {
+  translationNamespaces?: string[] | string;
+  testId: string;
+  className: string;
+  modalComponent: ReactNode;
+  legendTranslationKey: string;
+};
+
+const AddNewCard = ({
```

Comment:
This is a component specific to operational studies, maybe should be in a different directory? Maybe `src/applications/operationalStudies/components/`?